modules/items/client/views/list-items.client.view.html

```diff
@@ -26,6 +26,8 @@ <h4 class="list-group-item-heading" data-ng-bind="item.title"></h4>
       <span ng-if="item.modules" ng-bind="item.modules.title"></span>
       <b> Status:</b>
       <span ng-if="item.workingStatus" data-ng-bind="item.workingStatus"></span>
+      <b>PDF link: </b>
```

Comment:
I think we should either make the link part of the individual item view, or at least not display the header for the pdf link unless there actually is one. Move that inside the ng-if